bin/extract-release-notes.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+###############################################################################
+#
+# A bash script to extract only the notes related to the most recent version of
+# earthdata-varinfo from CHANGELOG.md
+#
+# 2023-06-16: Created
+#
+###############################################################################
+
+CHANGELOG_FILE="CHANGELOG.md"
+VERSION_PATTERN="^## v"
+# Count number of versions in version file:
+number_of_versions=$(grep -c "${VERSION_PATTERN}" ${CHANGELOG_FILE})
+
+if [ ${number_of_versions} -gt 1 ]
+then
+	grep -B 9999 -m2 "${VERSION_PATTERN}" ${CHANGELOG_FILE} | sed '$d' | sed '$d'
```

Comment:
Does -m2 need a space?  I think technically it doesn't need one, but it works with one.

Reply:
I can update to add the space - it maybe makes the flag a bit clearer.